docs/blog/2019-04-19-gatsby-why-we-write/index.md

```diff
@@ -0,0 +1,123 @@
+[Kyle](/contributors/kyle-mathews/),
+
+And it's not just the Inkteam. The documentation, while curated by the Inkteam, has largely been authored by enthusiastic community members, along with about half of our blog posts.
+
+Off this blog, on Twitter and the wider web, every day brings four or five community members writing about their experience.
+
+Why do we write so much?
+
+The answer:
+
+### Principle 2: content acts as a guide unlocking the power of Gatsby.
+
+Gatsby is so powerful, because of its core _and the ecosystem_ — not just plugins and themes, but React, Webpack and Babel, the entire npm ecosystem, and modern Javascript in general.
+
+You can do almost anything you want with Gatsby, you just have to:
+
+_(1) know why & that it's possible_
+
+_(2) know how to do it_
+
+Roughly speaking, there are two sides of content at Gatsby — marketing content which focuses on (1), and learning content which focuses on (2).
+
+An example of (1) might be Linda's [How To Talk About Gatsby To Your Clients and Team](https://www.gatsbyjs.org/blog/2019-03-07-sell-gatsby-to-clients/).
+
+An example of (2) might be Jason's [Learn With Jason](https://www.youtube.com/playlist?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx) series or really anything in the [docs](/docs/).
+
+There's tons of overlap - _why_ points to _how_, and *how *points to _why_. You could argue we've spent more time on (2) than (1), but the reality is that _we've only scratched the surface of what's possible_.
+
+### Principle 3: because Gatsby is so powerful, the ground to cover with writing is huge, yet essential
+
+- Marcy's Q2 2019 project is to catalogue the **[Top 25 Workflows](https://docs.google.com/presentation/d/1YJ_4NGwxPqtMo-kyWBmg7NBr_YYZezKZ1phXjl0abD4/edit#slide=id.gcb9a0b074_1_0)** people do with Gatsby and improve the experience for 5-10 of them. You'll notice that many of them are not exclusive to Gatsby, instead they're "what you need to do in order to build a website".
+
+- We identified about 50 key messages relating Gatsby to _high-level benefits_ (such as improving security, reducing cost, simplifying your stack), to _different technologies_ (Wordpress, Drupal), _vendors_ (Contentful, AWS), and _communities_(JAMStack, modern development, accessibility). In addition, they cover messages that are important for the wide range of personas in the Gatsby ecosystem -- marketers, educators, open-source contributors, plugin authors, numerous flavors of developers, etc.
+
+Each of these 70-80 messages & workflows probably has ~10 sub-workflows/sub-messages/features — imagine for each of these we had 3-5 case studies, docs pages, dedicated tutorials, starters/themes with annotated open-source source code -- that's thousands, probably tens of thousands, of pieces of content. _That's why we've only scratched the surface of what's possible._
+
+_(Now imagine a UI on top and the ability to mix & match!)_
```

Comment:
What is this referring to? Would love to hear more!

Reply:
Deliberately being a bit cryptic here :) this could be eg a way to track where you are in a tutorial as well as auto-select and combine starters / themes programatically via UI. But don't want to go too far here as might be a bit distracting.

Reply:
I personally think it would be less distracting without it, especially since it alludes to something that doesn't exist yet. What if we don't end up solving for that exact thing?

Reply:
Reasonable!